pkg/scheduler/core/generic_scheduler.go

```diff
@@ -182,13 +182,13 @@ func (g *genericScheduler) assignReplicas(
 		return nil, fmt.Errorf("no clusters available to schedule")
 	}
 
-	if object.Replicas > 0 && replicaSchedulingStrategy != nil {
```

Comment:
In fact, it's a bug. I think it needs to be cherry-picked to the previous branches. When `replicaSchedulingStrategy` is undefined, the workload will not be assigned replicas.

Reply:
Before this PR, when `replicaSchedulingStrategy` is undefined, it implies that `replicaSchedulingStrategy` is `duplicated`. Though `resourceBinding` do not show `replicas`, Karmada will propagate the workload with the whole replicas to member clusters. I wonder if it will affect your business.

Reply:
Thanks your reminder.